### PR TITLE
Improving the MicroProfile JWT sample with a configuration shell script

### LINKED ISSUE #3162: Add a sample to demonstrate the usage of MicroProfile JWT
Create a sample to demonstrate the usage of MicroProfile JWT compatible JWT token as mentioned in https://docs.wso2.com/display/IS560/Microprofile+JWT+1.0+support+for+IS

### PR DESCRIPTION
**Pull request by madawas**
Introducing a shell script to configure the assets needed to run the sample. Users, Roles, SP etc. And fixing an issue with the maven war file not mentioned version.

Resolves #3162

### CODE REVIEW

File: modules/samples/microprofile/microprofile-jwt/src/main/resources/assembly/dist.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        <fileSet>
+            <directory>${project.basedir}/src/main/resources</directory>
+            <outputDirectory>microprofile-jwt-resources</outputDirectory>
+            <useDefaultExcludes>true</useDefaultExcludes>
+            <excludes>
+                <exclude>assembly/**</exclude>
+            </excludes>
+        </fileSet>
+    </fileSets>
+</assembly>
```
Comment (madurangasiriwardena) [inline modules/samples/microprofile/microprofile-jwt/src/main/resources/assembly/dist.xml]: Add new line

File: modules/samples/microprofile/microprofile-jwt/src/main/resources/configs/get-sp.xml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:xsd="http://org.apache.axis2/xsd">
+    <soapenv:Header/>
+    <soapenv:Body>
+        <xsd:getApplication>
+            <!--Optional:-->
+            <xsd:applicationName>microprofile_jwt_sample</xsd:applicationName>
+        </xsd:getApplication>
+    </soapenv:Body>
+</soapenv:Envelope>
```
Comment (madurangasiriwardena) [inline modules/samples/microprofile/microprofile-jwt/src/main/resources/configs/get-sp.xml]: Add new line

File: modules/samples/microprofile/microprofile-jwt/src/main/resources/assembly/dist.xml (hunk 1 above)
Comment (madawas) [reply modules/samples/microprofile/microprofile-jwt/src/main/resources/assembly/dist.xml]: Fixed with dd90d2d

File: modules/samples/microprofile/microprofile-jwt/src/main/resources/configs/get-sp.xml (hunk 2 above)
Comment (madawas) [reply modules/samples/microprofile/microprofile-jwt/src/main/resources/configs/get-sp.xml]: Fixed with dd90d2d

File: modules/samples/microprofile/microprofile-jwt/src/main/resources/configure_sample.sh [hunk 3]
```diff
@@ -0,0 +1,523 @@
...
+    echo "!! Problem occurred while creating user cameron. !!"
+    echo
+    return 255
+  fi
+  echo "** The user cameron was successfully created. **"
+  echo
+
+  echo "Creating a user named alex..."
+
+  curl -s -k --user "${IS_name}":"${IS_pass}" --data '{"schemas":[],"name":{"familyName":"Miller","givenName":"Alex"},"userName":"alex","password":"alex123","emails":"alex@gmail.com","addresses":{"country":"United States"}}' --header "Content-Type:application/json" -o /dev/null https://localhost:9443/wso2/scim/Users
```
Comment (pulasthi7) [inline modules/samples/microprofile/microprofile-jwt/src/main/resources/configure_sample.sh]: let's change the mail to some dummy one later..

Comment (madawas) [reply modules/samples/microprofile/microprofile-jwt/src/main/resources/configure_sample.sh]: +1

**Pull request merged**